### DISCUSSION TITLE
Support for Windows

### DISCUSSION THREAD

**Discussion by waleedrana777** [firebase/flutterfire#11469] [Feature request]
That's it, support firebase_auth and firebase_firestore on windows, and that is really sad, after so many years, still!

**Comment by hwkim1127** [comment]
Well I think we should keep asking Google for it

Here
https://github.com/firebase/firebase-cpp-sdk/issues/442

Here
https://firebase.uservoice.com/forums/948424-general/suggestions/46591750-desktop-windows-linux-macos-support-for-flutte

And here
https://firebase.google.com/support/troubleshooter/report/features

cuz personally I think it's not what flutter team can do unless Google Cloud and Firebase team actually works on it.
